_posts/2017-07-09-P3-Quality.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+asset-type: post
+name: P3 Quality
+title: 'P3 Quality'
+date: 2017-07-09 11:00:00 +00:00
+author: Raquel M Carmena
+image:
+  src: /assets/img/custom/blog/2017-07-09-P3-Quality.jpg
+tags:
+- craftsmanship
+- quality
+
+---
+
+Do we know what is software quality? Is it measurable? If we want to improve the quality, do we know what to focus on?
+
+Nowadays, it's still thought that quality is having automated functional tests. A manager worried about quality tends to hire QA-roles or to contract of an external company to automate functional tests.
+
+On the other hand, standards ISO/IEC 250xx provide a lot of metrics and guides to measure the quality of a product, but is that enough? What about the process that is followed to create that product? 
+
+Others think that they must embrace agile methodologies, but a lot of agile adoptions fail, because they only focus on the process, not on technical excellence.
+
+That’s the reason why we need a global concept of quality, that we’ll call **P3 QUALITY**. 
+
+If we want to ensure or improve quality, we should focus on:
+
+* PROCESS
+* PRODUCT
+* PEOPLE
+
+where the most important part is **PEOPLE**.
+
+<center>
+<img src="{{site.baseurl}}/assets/img/custom/blog/2017-07-09-P3-Quality/P3-Quality.jpg" alt="P3 Quality" class="img img-responsive"/>
+<small>P3 QUALITY scheme</small>
+</center>
+
+<br/>
+
+When reading the book <a href="https://www.goodreads.com/book/show/23215733-the-software-craftsman" target="_blank">**The Software Craftsman. Professionalism, Pragmatism, Pride** by Sandro Mancuso</a>, you notice that the problems in software projects are the same, regardless of the company or the country, and the key to success is in the people’s quality, in our skills and attitude. 
```

Comment:
It would be good to have more sources of software literacy where we understand that it is always a "people problem"